templates/ever-teams/meta.yaml

```diff
@@ -0,0 +1,104 @@
+name: Ever Teams
+description: Ever Teams is a powerful and flexible work and project management
+  platform that can help teams of all sizes to improve their productivity,
+  collaboration, and visibility. It is also highly customizable and can be integrated
+  with a variety of third-party tools. If you are looking for a work and project management
+  platform that can help your team to achieve its goals, then Ever Teams is a great option.
+instructions: null
+changeLog:
+  - date: 2023-11-16
+    description: first release
+links:
+  - label: Github
+    url: https://github.com/ever-co/ever-teams
+contributors:
+  - name: paradoxe35
```

Comment:
@paradoxe35 please change this to:
- name: Ever Co. LTD
- url: https://github.com/ever-co